pkg/cli/admin/release/info.go

```diff
@@ -1165,10 +1165,101 @@ func describeReleaseDiff(out io.Writer, diff *ReleaseDiff, showCommit bool, outp
 			}
 		})
 	}
+
+	printFeatureSetSection(diff, "Default", w)
+	printFeatureSetSection(diff, string(configv1.TechPreviewNoUpgrade), w)
+
 	fmt.Fprintln(w)
 	return nil
 }
 
+func printFeatureSetSection(diff *ReleaseDiff, featureSetName string, w io.Writer) {
+	newFeatures, newlyEnabledFeatures, newlyDisabledFeatures, newlyRemovedFeatures, _ := featureGatesForFeatureSetChanged(diff, featureSetName)
+	if newFeatures == nil {
+		return
+	}
+	fmt.Fprintln(w)
+	fmt.Fprintf(w, "FeatureGates for %q (%d enabled, %d disabled, %d removed)\n",
+		featureSetName, len(newlyEnabledFeatures), len(newlyDisabledFeatures), len(newlyRemovedFeatures))
+	fmt.Fprintf(w, "Enabled:\n")
+	writeTabSection(w, func(w io.Writer) {
+		for _, featureGate := range newFeatures.Status.FeatureGates[0].Enabled {
+			if newlyEnabledFeatures.Has(featureGate.Name) {
+				fmt.Fprintf(w, "*%v\n", featureGate.Name)
+			} else {
+				fmt.Fprintf(w, "%v\n", featureGate.Name)
+			}
+		}
+	})
+	fmt.Fprintf(w, "Disabled:\n")
+	writeTabSection(w, func(w io.Writer) {
+		for _, featureGate := range newFeatures.Status.FeatureGates[0].Disabled {
+			if newlyDisabledFeatures.Has(featureGate.Name) {
+				fmt.Fprintf(w, "*%v\n", featureGate.Name)
+			} else {
+				fmt.Fprintf(w, "%v\n", featureGate.Name)
+			}
+		}
+	})
+	if len(newlyRemovedFeatures) > 0 {
+		fmt.Fprintf(w, "Removed:\n")
+		writeTabSection(w, func(w io.Writer) {
+			for _, feature := range newlyRemovedFeatures {
+				fmt.Fprintf(w, "%v\n", feature)
+			}
+		})
+	}
+}
+
+// featureGatesForFeatureSetChanged returns newly enabled featureGates, newly disabled featureGates, and newly removed featureGates
+func featureGatesForFeatureSetChanged(diff *ReleaseDiff, featureSetName string) (*configv1.FeatureGate, sets.Set[configv1.FeatureGateName], sets.Set[configv1.FeatureGateName], sets.Set[configv1.FeatureGateName], bool) {
+	manifestName := fmt.Sprintf("0000_50_cluster-config-api_featureGate-%v.yaml", featureSetName)
+	changed := diff.ChangedManifests[manifestName]
```

Comment:
This would panic against an old release payload?

Reply:
Next link checks for nil to avoid the panic.

Reply:
I meant the cases where manifestname does not exist

Reply:
> I meant the cases where manifestname does not exist

Yeah, in those cases changed is nil.